privilege/privileges/cache.go

```diff
@@ -296,6 +293,10 @@ func (p *MySQLPrivilege) loadTable(sctx sessionctx.Context, sql string,
 				return errors.Trace(err)
 			}
 		}
+		// NOTE: decodeTableRow decodes data from a chunk Row, that is a shallow copy.
```

Comment:
Please read the NOTE once again! @lysu

Reply:
yes, in here, use `Renew` will always create a new chunk just like old logic :laughing:

Reply:
ok